go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/elastic/go-ucfg v0.8.4
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/magefile/mage v1.12.1
+	github.com/pkg/errors v0.9.1
```

Comment:
I've been asked to not used this deprecated package anymore, see https://github.com/elastic/elastic-agent-libs/pull/10#discussion_r808796031